src/neoxp/Commands/TransferNFTCommand.cs

```diff
@@ -0,0 +1,67 @@
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+using McMaster.Extensions.CommandLineUtils;
+
+namespace NeoExpress.Commands
+{
+    [Command("transfernft", Description = "Transfer NFT asset between accounts")]
+    class TransferNFTCommand
+    {
+        readonly ExpressChainManagerFactory chainManagerFactory;
+        readonly TransactionExecutorFactory txExecutorFactory;
+
+        public TransferNFTCommand(ExpressChainManagerFactory chainManagerFactory, TransactionExecutorFactory txExecutorFactory)
+        {
+            this.chainManagerFactory = chainManagerFactory;
+            this.txExecutorFactory = txExecutorFactory;
+        }
+
+        [Argument(0, Description = "NFT Contract (symbol or script hash)")]
+        [Required]
+        internal string Contract { get; init; } = string.Empty;
+
+        [Argument(1, Description = "TokenId of NFT (Base64 string)")]
+        [Required]
+        internal string TokenId { get; init; } = string.Empty;
+
+        [Argument(2, Description = "Account to send NFT from")]
+        [Required]
+        internal string Sender { get; init; } = string.Empty;
+
+        [Argument(3, Description = "Account to send NFT to")]
+        [Required]
+        internal string Receiver { get; init; } = string.Empty;
+
+        [Option(Description = "Optional data parameter to pass to transfer operation")]
+        internal string Data { get; init; } = string.Empty;
+
+        [Option(Description = "password to use for NEP-2/NEP-6 sender")]
+        internal string Password { get; init; } = string.Empty;
```

Comment:
`NEP-17` transfers dont require password. Seeing how there is `default.neo-express` file that holds it.

Reply:
This is copied from TransferCommand.cs and is the same as NEP-17.

Reply:
yes but for `password to use for NEP-2/NEP-6 sender` for legacy i believe.

Reply:
Also keep in mind for `batch` it doesn't require password for transfers. That's what i was thinking of.